sdk/python/foundation-models/system/finetune/text-to-image/hftransformers-dreambooth-dog-text-to-image.ipynb

```diff
@@ -0,0 +1,704 @@
+    "workspace_ml_client = MLClient(credential, subscription_id, resource_group, workspace_name)\n",
+    "registry_ml_client = MLClient(\n",
+    "    credential,\n",
+    "    subscription_id,\n",
+    "    resource_group,\n",
+    "    registry_name=\"azureml\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2.2 Create compute\n",
+    "\n",
+    "In order to finetune a model on Azure Machine Learning studio, you will need to create a compute resource first. **Creating a compute will take 3-4 minutes.** \n",
+    "\n",
+    "For additional references, see [Azure Machine Learning in a Day](https://github.com/Azure/azureml-examples/blob/main/tutorials/azureml-in-a-day/azureml-in-a-day.ipynb). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import AmlCompute\n",
+    "from azure.core.exceptions import ResourceNotFoundError\n",
+    "\n",
+    "cluster_name = \"sample-finetune-cluster-gpu\"\n",
+    "\n",
+    "try:\n",
+    "    _ = workspace_ml_client.compute.get(cluster_name)\n",
+    "    print(\"Found existing compute target.\")\n",
+    "except ResourceNotFoundError:\n",
+    "    print(\"Creating a new compute target...\")\n",
+    "    compute_config = AmlCompute(\n",
+    "        name=cluster_name,\n",
+    "        type=\"amlcompute\",\n",
+    "        size=\"Standard_NC6s_v3\",\n",
+    "        idle_time_before_scale_down=120,\n",
+    "        min_instances=0,\n",
+    "        max_instances=4,\n",
+    "    )\n",
+    "    workspace_ml_client.begin_create_or_update(compute_config).result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Pick a foundation model to fine tune\n",
+    "\n",
+    "We will use the `runwayml-stable-diffusion-v1-5` model in this notebook. If you need to fine tune a model that is available on HuggingFace, but not available in `azureml` system registry, you can either register the model and use the registered model or use the `model_name` parameter to instruct the components to pull the model directly from HuggingFace.\n",
+    "\n",
+    "Currently following models are supported:\n",
+    "\n",
+    "| Model Name | Source |\n",
+    "| ------ | ---------- |\n",
+    "| [runwayml-stable-diffusion-v1-5](https://ml.azure.com/registries/azureml/models/runwayml-stable-diffusion-v1-5/version/8) | azureml registry |\n",
+    "| [stabilityai-stable-diffusion-2-1](https://ml.azure.com/registries/azureml/models/stabilityai-stable-diffusion-2-1/version/8) | azureml registry |\n",
+    "| [compvis-stable-diffusion-v1-4](https://ml.azure.com/registries/azureml/models/compvis-stable-diffusion-v1-4/version/8) | azureml registry |\n",
+    "| [Text to Image models from Huggingface's Transformer library](https://huggingface.co/models?pipeline_tag=text-to-image&library=transformers)| HuggingFace |"
```

Comment:
deci diffusion model works as well. Please add that to the list. 
Sample run: https://ml.azure.com/runs/8d654f9f-1ac1-4398-9ad1-ea9f4e788b26?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/grajguru-rg/providers/Microsoft.MachineLearningServices/workspaces/grajguru-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Reply:
Added